pkg/api/v1alpha2/types.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type DeschedulerPolicy struct {
```

Comment:
This does not correspond to v1alpha2 as specified in https://docs.google.com/document/d/1S1JCh-0F-QCJvBBG-kbmXiHAJFF8doArhDIAKbOj93I/edit#. I am expecting:
```
type DeschedulerPolicy struct {
  metav1.TypeMeta `json:”,inline”`

  Profiles []DeschedulerProfile `json:”profiles”`
  ...
}

type DeschedulerProfile struct {
  Name       string `json:”name”`
  Strategies StrategyList `json:”strategies”`
}
```
@hanumanthan any reason you did not incorporate the new type in the first commit?

Reply:
I seperated as two commits to 
1. Make review easier for this [file](pkg/descheduler/policyconfig.go)
2. To verify if the type conversion happens.

Would you like two commits to be merged as one ?
> This does not correspond to v1alpha2 as specified

Are you highlighting this in the context of first commit or this PR (confirming if the 2nd commit matches the design spec)?

Reply:
Given the first commit is named "Adding v1alpha2 version with backward version compatibility" I'd assume you define the v1alpha2 version of the type properly there.